package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "contentful-ui-extensions-sdk",
   "description": "SDK to develop custom UI Extension for the Contentful Web App",
-  "version": "3.14.3",
+  "version": "3.15.0",
```

Comment:
There should be a change in `package-lock.json` too.

Reply:
Sorry slipped through :). Good catch!